tests/ProcessMaker/Repositories/BpmnFileRepository.php

```diff
@@ -43,6 +44,8 @@ class BpmnFileRepository extends DOMDocument implements RepositoryFactoryInterfa
      */
     private $engine;
 
+    private $executionInstanceRepository;
```

Comment:
Add the documentation to this private attribute, similar to the rest of the attributes